### PR TITLE
Update snowflake-setup.md

### PR DESCRIPTION
**Pull request by sophiad96**
Add link to specific Snowflake docs section that determines whether region and cloud platform need to be included.

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/snowflake-setup.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
...
+For AWS accounts in the US West default region, you can use `abc123` (without any other segments). For some AWS accounts you will have to append the region and/or cloud platform. For example, `abc123.eu-west-1` or `abc123.eu-west-2.aws`. For GCP and Azure-based accounts, you have to append the region and cloud platform, such as `gcp` or `azure`, respectively. For example, `abc123.us-central1.gcp`. For details, see Snowflake's documentation: "[Specifying Region Information in Your Account Hostname](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname)". Please also note that the Snowflake account name should only be the <account_name> without the prefixed <organization_name>.  To determine if the region and/or cloud platform needs to be appended to the account locator in the legacy format, see Snowflake's documentation on "[Non-VPS account locator formats by cloud platform and region](https://docs.snowflake.com/en/user-guide/admin-account-identifier#non-vps-account-locator-formats-by-cloud-platform-and-region)".
```
Comment (runleonarun) [inline website/docs/docs/core/connect-data-platform/snowflake-setup.md]:
```suggestion
For AWS accounts in the US West default region, you can use `abc123` (without any other segments). For some AWS accounts you will have to append the region and/or cloud platform. For example, `abc123.eu-west-1` or `abc123.eu-west-2.aws`. 

For GCP and Azure-based accounts, you have to append the region and cloud platform, such as `gcp` or `azure`, respectively. For example, `abc123.us-central1.gcp`. For details, see Snowflake's documentation: "[Specifying Region Information in Your Account Hostname](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname)". 

Please also note that the Snowflake account name should only be the <account_name> without the prefixed <organization_name>.  To determine if the region and/or cloud platform needs to be appended to the account locator in the legacy format, see Snowflake's documentation on "[Non-VPS account locator formats by cloud platform and region](https://docs.snowflake.com/en/user-guide/admin-account-identifier#non-vps-account-locator-formats-by-cloud-platform-and-region)".
```

**Review by runleonarun — COMMENTED** [review]
Thanks for adding this info @sophiad96! I just split it up so it's easier to parse and will merge this soon!

File: website/docs/docs/core/connect-data-platform/snowflake-setup.md [hunk 2]
```diff
@@ -247,7 +247,11 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
...
+For AWS accounts in the US West default region, you can use `abc123` (without any other segments). For some AWS accounts you will have to append the region and/or cloud platform. For example, `abc123.eu-west-1` or `abc123.eu-west-2.aws`. 
+
+For GCP and Azure-based accounts, you have to append the region and cloud platform, such as `gcp` or `azure`, respectively. For example, `abc123.us-central1.gcp`. For details, see Snowflake's documentation: "[Specifying Region Information in Your Account Hostname](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname)". 
+
+Please also note that the Snowflake account name should only be the <account_name> without the prefixed <organization_name>.  To determine if the region and/or cloud platform needs to be appended to the account locator in the legacy format, see Snowflake's documentation on "[Non-VPS account locator formats by cloud platform and region](https://docs.snowflake.com/en/user-guide/admin-account-identifier#non-vps-account-locator-formats-by-cloud-platform-and-region)".
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/snowflake-setup.md]:
```suggestion
Please also note that the Snowflake account name should only be the `account_name` without the prefixed `organization_name`.  To determine if the region and/or cloud platform needs to be appended to the account locator in the legacy format, see Snowflake's documentation on "[Non-VPS account locator formats by cloud platform and region](https://docs.snowflake.com/en/user-guide/admin-account-identifier#non-vps-account-locator-formats-by-cloud-platform-and-region)".
```

**Review by mirnawong1 — APPROVED** [review]
lgtm, merging for ya'll!

**Pull request merged**